helix-term/src/commands/lsp.rs

```diff
@@ -831,6 +947,17 @@ pub fn signature_help_impl(cx: &mut Context, invoked: SignatureHelpInvoked) {
             return;
         }
     };
+    signature_help_impl_with_language_server_id(cx, language_server_id, invoked);
+}
+
+pub fn signature_help_impl_with_language_server_id(
+    cx: &mut Context,
+    language_server_id: usize,
+    invoked: SignatureHelpInvoked,
+) {
+    let was_manually_invoked = invoked == SignatureHelpInvoked::Manual;
```

Comment:
I don't think this is being used anywhere?

Reply:
What do you mean exactly? It's being used for example here: https://github.com/Philipp-M/helix/blob/090a0f4d31a4c1d94c8ce10e353266a2309c55f9/helix-term/src/commands/lsp.rs#L977
But I can remove the variable and use `invoked == SignatureHelpInvoked::Manual` directly there though.

Reply:
Ah sorry, apparently I can't `Ctrl-F`. I'd have a slight preference for just using the value, as I don't think it makes any difference in readability, but I have no strong preference.